docs/conf.py

```diff
@@ -287,14 +287,13 @@
     "python": ("https://docs.python.org/3", None),
     "numpy": ("https://numpy.org/doc/stable", None),
     "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
-    "torch": ("https://pytorch.org/docs/master/", None),
+    "torch": ("https://pytorch.org/docs/main/", None),
     "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
     "matplotlib": ("https://matplotlib.org", None),
     "pandera": ("https://pandera.readthedocs.io/en/stable/", None),
 }
 
 myst_enable_extensions = ["colon_fence"]
-myst_heading_anchors = 6
```

Comment:
This should not be deleted -- this is the heading level depth to assign HTML anchors. (The default is zero, so if removed, I think we lose HTML anchors.)

Reply:
Ah sorry, this was a mistake. Reverted.